docs/software/lumi-o/error-messages.md

```diff
@@ -0,0 +1,36 @@
+# Common error messages
```

Comment:
I might add also status code 503
Since in case system is suffering resources in backend, QoS will slow down operations